conf.d/health_alarm_notify.conf

```diff
@@ -61,6 +61,20 @@ curl=""
 # If not found, irc notifications will be silently disabled.
 nc=""
 
+#------------------------------------------------------------------------------
+# extra options for external commands
+#
+# In some cases, you may need to change what options get passed to an
+# external command.  Such cases are covered here.
+
+# Extra options to pass to curl.  In most cases, you shouldn't need to add anything
+# to this.  If you're having issues with HTTPS connections, you might try adding
+# '--insecure' here, but be warned that it will make it much easier for
+# third-parties to block notification delivery, and may allow disclosure
+# of potentially sensitive information.
+curl_options=""
```

Comment:
perfect!
Please move this to the script, so that if the config is missing, the script will not inherit something from the shell.

Reply:
Actually, the line could in theory just be completely removed.  Undefined variables expand to an empty string in all POSIX compliant shells including bash, and the default value should be an empty string.

Reply:
ok, I'll do it.